Hi @Ninja91 this is the fails we see,

[pull / unittest-arm-backend-with-no-fvp (test_pytest_ops) / linux-job (pull_request)](https://github.com/pytorch/executorch/actions/runs/19161380882/job/54772221446?pr=15631)
FAILED backends/arm/test/ops/test_rsqrt.py::test_rsqrt_int16_tosa_INT[rand_4d_1]
FAILED backends/arm/test/ops/test_rsqrt.py::test_rsqrt_int16_tosa_INT[rand_4d_2]
FAILED backends/arm/test/ops/test_rsqrt.py::test_rsqrt_int16_tosa_INT[rand_3d]

 if you can break out the fails in it's own tests you can add a tag 
@pytest.mark.xfail( reason="text" )
for now, or just commen them out.

--- a/backends/arm/test/ops/test_rsqrt.py
+++ b/backends/arm/test/ops/test_rsqrt.py
@@ -8,9 +8,14 @@
 
 from typing import Tuple
 
+import pytest
 import torch
+from executorch.backends.arm.quantizer.arm_quantizer import (
+    get_symmetric_a16w8_quantization_config,
+    TOSAQuantizer,
+)
+from executorch.backends.arm.test import common, conftest
 
-from executorch.backends.arm.test import common
 from executorch.backends.arm.test.tester.test_pipeline import (
     EthosU55PipelineINT,
     EthosU85PipelineINT,
@@ -18,7 +23,8 @@ from executorch.backends.arm.test.tester.test_pipeline import (
     TosaPipelineINT,
     VgfPipeline,
 )
-
+from executorch.backends.arm.tosa import TosaSpecification
+from executorch.backends.xnnpack.test.tester import Quantize
 
 aten_op = "torch.ops.aten.rsqrt.default"
 input_t1 = Tuple[torch.Tensor]  # Input x
@@ -102,5 +108,101 @@ def test_rsqrt_vgf_INT(test_tensor: torch.Tensor):
         test_tensor(),
         aten_op,
         tosa_version="TOSA-1.0+INT",
+    )
+    pipeline.run()
+
+
+def get_symmetric_a16w8_rsqrt_quantizer(
+    u55_config=False, per_channel_quantization=False
+):
+    tosa_version = conftest.get_option("tosa_version")
+    tosa_profiles = {
+        "1.0": TosaSpecification.create_from_string("TOSA-1.0+INT+int16"),
+    }
+
+    quantizer = TOSAQuantizer(tosa_profiles[tosa_version])
+    quantizer.set_global(
+        get_symmetric_a16w8_quantization_config(is_per_channel=per_channel_quantization)
+    )
+
+    return Quantize(
+        quantizer,
+        get_symmetric_a16w8_quantization_config(
+            is_per_channel=per_channel_quantization
+        ),
+    )
+
+
+@common.parametrize("test_tensor", Rsqrt.test_parameters)
+@pytest.mark.xfail(
+    reason="MLETORCH-707: AssertionError: Output 0 does not match reference output."
+)
+def test_rsqrt_16a8w_tosa_INT(test_tensor: torch.Tensor):
+    """Test rsqrt operation with int16 quantization"""
+    pipeline = TosaPipelineINT[input_t1](
+        Rsqrt(),
+        test_tensor(),
+        aten_op,
+        exir_op=[],
+        per_channel_quantization=False,
+        use_to_edge_transform_and_lower=True,
+        tosa_extensions=["int16"],
+    )
+
+    pipeline.change_args(
+        "quantize",
+        get_symmetric_a16w8_rsqrt_quantizer(per_channel_quantization=False),
+    )
+    # Run the pipeline
+    pipeline.run()
+
+
+@common.parametrize("test_tensor", Rsqrt.test_parameters)
+@common.XfailIfNoCorstone300
+@pytest.mark.xfail(
+    reason="MLETORCH-707: AssertionError: Output 0 does not match reference output."
+)
+def test_rsqrt_16a8w_u55_INT16(test_tensor: torch.Tensor):
+    """Test rsqrt operation with int16 quantization on U55"""
+    pipeline = EthosU55PipelineINT[input_t1](
+        Rsqrt(),
+        test_tensor(),
+        aten_op,
+        exir_ops=[],
+        per_channel_quantization=True,
+        use_to_edge_transform_and_lower=True,
+        atol=1e-03,
+        rtol=1e-03,
+        run_on_fvp=True,
+    )
+
+    pipeline.change_args(
+        "quantize",
+        get_symmetric_a16w8_rsqrt_quantizer(per_channel_quantization=True),
+    )
+    pipeline.run()
+
+
+@common.parametrize("test_tensor", Rsqrt.test_parameters)
+@common.XfailIfNoCorstone320
+@pytest.mark.xfail(
+    reason="MLETORCH-707: AssertionError: Output 0 does not match reference output."
+)
+def test_rsqrt_16a8w_u85_INT16(test_tensor: torch.Tensor):
+    """Test rsqrt operation with int16 quantization on U85"""
+    pipeline = EthosU85PipelineINT[input_t1](
+        Rsqrt(),
+        test_tensor(),
+        aten_op,
+        exir_ops=[],
+        use_to_edge_transform_and_lower=True,
+        atol=1e-03,
+        rtol=1e-03,
+        run_on_fvp=True,
+    )
+
+    pipeline.change_args(
+        "quantize",
+        get_symmetric_a16w8_rsqrt_quantizer(per_channel_quantization=False),
     )
     pipeline.run()

--- a/backends/arm/test/targets.bzl
+++ b/backends/arm/test/targets.bzl
@@ -22,6 +22,7 @@ def define_arm_tests():
         "ops/test_linear.py", 
         "ops/test_mul.py",
         "ops/test_permute.py",
+        "ops/test_rsqrt.py",
         "ops/test_slice.py",
         "ops/test_sigmoid.py",
         "ops/test_sub.py",

